BoxContentSDK/BoxContentSDK/OAuth2/BOXAuthorizationViewController.m

```diff
@@ -168,35 +168,44 @@ - (void)viewDidAppear:(BOOL)animated
                                      message:self.connectionErrorMessage];
     }
 
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(applicationWillResignActive:) name:UIApplicationWillResignActiveNotification object:nil];
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(applicationDidBecomeActive:) name:UIApplicationDidBecomeActiveNotification object:nil];
+    [[NSNotificationCenter defaultCenter] addObserver:self
+                                             selector:@selector(didReceiveBackgroundNotification:)
+                                                 name:UIApplicationDidEnterBackgroundNotification
```

Comment:
This is not too late?

Reply:
we don't get UIApplicationDidBecomeActiveNotification at all. Verified this works on iOS 11.3 device.

Reply:
Apple docs suggest it's okay. https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622997-applicationdidenterbackground?language=objc